cont/base/springcontent/LuaGadgets/Gadgets/share_delayed.lua

```diff
@@ -400,3 +389,13 @@ end
 --------------------------------------------------------------------------------
 --  COMMON
 --------------------------------------------------------------------------------
+
+function gadget:CommandNotify(cmdID, cmdParams, cmdOptions, playerID)
+  if (cmdID == CMD_CANCEL_SHARE) then
+    for _,unitID in ipairs(Spring.GetSelectedUnits(playerID)) do
+      RemoveShare(unitID)
```

Comment:
`CMD_CANCEL_SHARE` and `RemoveShare` aren't added in this PR but that's also fine because they weren't in engine `/take`, remove these.